src/tenderly/run-all-flows.ts

```diff
@@ -0,0 +1,48 @@
+import * as hre from "hardhat";
+import { deployVerifyZNS } from "./deploy-verify-zns";
+import * as ethers from "ethers";
+import { hashDomainLabel } from "../../test/helpers";
+import { BigNumber } from "ethers";
+
+
+const domainName = "wilder";
+const domainHash = hashDomainLabel(domainName);
+const tokenId = BigNumber.from(domainHash);
```

Comment:
If the `tokenId` internally is the `uint256(domainHash)` then won't this not work? This is just going to be the BigNumber of that `bytes32` object?

Reply:
not sure what you mean here. bignumber is casting number into a bytes hash. when we use it as a number it converts that to an int. why would this not work?

Reply:
Did you confirm this works? It wouldn't be an `int` yet explicitly as you haven't called to modify it and it would just be "0x..." internally still. I guess it just does this implicitly in the function call below?